ingestion-beam/src/main/java/com/mozilla/telemetry/Sink.java

```diff
@@ -67,4 +77,60 @@ static void registerOptions() {
     PipelineOptionsFactory.register(DecoderOptions.class);
   }
 
+  static class FlexTemplateMetadata {
+
+    public static void main(String[] args) {
+      run(SinkOptions.class);
+    }
+
+    static void run(Class<? extends PipelineOptions> optionsClass) {
+      final ArrayNode parameters = Json.createArrayNode();
+      for (Method method : optionsClass.getMethods()) {
```

Comment:
I wasn't sure about using reflection here, but I looked up how the help page is generated and all the methods were private and use reflection, so I figure that's just what we get.

Reply:
Can you add a comment pointing to that help page code?

Reply:
the reflection stuff was removed because it's not required